.jules/runs/d657338a-caa9-4ccf-93a1-4733ada7154c/decision.md

```
@@ -0,0 +1 @@
+Migrated from legacy quality ledger.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)